addons/project/res_partner.py

```diff
@@ -36,6 +36,7 @@ def _task_count(self, cr, uid, ids, field_name, arg, context=None):
         'task_count': fields.function(_task_count, string='# Tasks', type='integer'),
     }
 
+    @api.cr_uid_id_context
```

Comment:
@NL66278  I think the two solutions are equivantent, that's what the @api.cr_uid_id_context does, it forces the triggering.  Maybe this is better because it relies on a decorator that explicity triggers v7API, looks more solid for future compatibility. (e.g. Let's say odoo drops automatic recognition of signature, this would hold)